app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -1,8 +1,8 @@
 <p>If you’re studying for a degree and interested in a career in teaching, an internship could help you to understand what it’s really like in the classroom.</p>
 
-<p> You’ll get to spend 3 weeks in a school, gain new skills, and be paid £300 per week.</p>
+<p> You’ll get the opportunity to experience a range of activities to help you get a feel for school life. Internships last 3 weeks, start in June and you’ll be paid £300 per week.  </p>
```

Comment:
I wonder if we'd be better splitting this paragraph so the first sentence goes into the para before and leaves the `Internships last 3 weeks.....` sentence as a standalone para as this is really critical info?